code/dictation.py

```diff
@@ -224,6 +225,19 @@ def format(self, text, auto_cap=True):
         self.before = text or self.before
         return text
 
+    # These are used as callbacks by prose modifiers / dictation_mode commands.
+    def cap(self): self.force_capitalization = "cap"
+    def no_cap(self): self.force_capitalization = "no cap"
+    def no_space(self):
+        # This is typically used after repositioning the cursor, so it is helpful to
+        # reset capitalization as well.
+        #
+        # FIXME: this sets state to "sentence start", capitalizing the next
```

Comment:
Totally open to changing/removing this. In my fork I default state to None, but I think the reasoning in my comment above is orthogonal. For example when positioning the cursor to a new text box, which is a common use of this for me, presumably folks would want to reset capitalization to whatever is their default.

Reply:
hm, I guess that's true. I'm mainly concerned not with the default dictation state but with the "no space" command, for which it feels weird that "foo no space bar" should produce "fooBar". But I guess you might want "no space hello" to produce "Hello" if you've just entered a new text box.